### ISSUE TITLE
Yet again issue with `apugli:modify_enchantment_level` not working properly

### ISSUE THREAD

**Issue by CyraLX** [MerchantPug/apugli#58]
With the example power it doesn't modify enchantment level if do not have a helmet on your head
I also observed this being weirdly inconsistent and working fine sometimes? And in some cases (?) on servers it also doesn't work regardless if helmet is present or not.

Example Power:
```json
{
    "type": "apugli:modify_enchantment_level",
    "enchantment": "minecraft:respiration",
    "condition": {
        "type": "origins:air",
        "comparison": ">",
        "compare_to": 1
    },
    "modifier": {
        "operation": "add_base_early",
        "value": 10
    }
}
```

**Comment by ChrysanthCow** [comment]
I forgot to tag this bug within the commit, but consider this fixed for next update.

**Comment by ChrysanthCow** [comment]
Fixed in 2.8.0
